src/api/hooks/Group.tsx

```diff
@@ -0,0 +1,23 @@
+import { useMutation, UseMutationResult, useQuery, useQueryClient } from 'react-query';
+import API from 'api/api';
+import { Group, RequestResponse } from 'types/Types';
+
+const QUERY_KEY_GROUPS = 'groups';
+const QUERY_KEY = 'group';
+export const useGroup = (slug: string) => {
+  return useQuery<Group>([QUERY_KEY, slug], () => API.getGroup(slug));
+};
+export const useUpdateGroup = (): UseMutationResult<Group, RequestResponse, Group, unknown> => {
+  const queryClient = useQueryClient();
+
+  return useMutation((group) => API.updateGroup(group), {
```

Comment:
Ref slug og Partial<Group> i api
```suggestion
export const useUpdateGroup = (slug: string): UseMutationResult<Group, RequestResponse, Partial<Group>, unknown> => {
  const queryClient = useQueryClient();

  return useMutation((group) => API.updateGroup(slug, group), {
```